docs/source/conf.py

```diff
@@ -145,55 +136,47 @@ def get_python_version_from_pyproject():
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-# html_theme = "sphinx_rtd_theme"
-# html_theme = "bizstyle"
-html_theme = "alabaster"
+html_theme = "sphinx_book_theme"
 
 html_title = f"{project} v{version} Manual"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
-    "description": f"Simulation Tools for CTA v{version}",
-    "github_user": "gammasim",
-    "github_repo": "simtools",
-    "fixed_sidebar": True,
-    "show_powered_by": False,
+    "path_to_docs": "docs",
+    "repository_url": "https://github.com/gammasim/simtools",
+    "repository_branch": "main",
+    "use_issues_button": True,
+    "show_toc_level": 1,
+    "announcement": (
+        "simtools is under rapid development with continuous changes. "
```

Comment:
This is repeating what is already in README.rst

Reply:
Yes, but this appears on top of the documentation on every page. I do think it is important enough to repeat it.

![Screenshot 2024-04-29 at 13 58 05](https://github.com/gammasim/simtools/assets/14001688/04f1a591-0b50-40f7-95fa-1d470a92af75)